src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -289,12 +289,12 @@ public static class Parser
                 new Regex(@"(?:.*(?:\""|^))(?<title>.*?)(?:[-_\W](?<![()\[]))+(?:\W?Season\W?)(?<season>(?<!\d+)\d{1,2}(?!\d+))(?:\W|_)+(?:Episode\W)(?:[-_. ]?(?<episode>(?<!\d+)\d{1,2}(?!\d+)))+",
                           RegexOptions.IgnoreCase | RegexOptions.Compiled),
 
-                // Multi-episode with episodes in square brackets (Series Title [S01E11E12] or Series Title [S01E11-12])
-                new Regex(@"(?:.*(?:^))(?<title>.*?)[-._ ]+\[S(?<season>(?<!\d+)\d{2}(?!\d+))(?:[E-]{1,2}(?<episode>(?<!\d+)\d{2}(?!\d+)))+\]",
```

Comment:
Is this change actually necessary? There are no related tests for it.

Instead of `\s` `[._ ]` would be better to handle spaces, periods or underscores and matches the format already used.

Reply:
I removed changes related to square brackets